### PR TITLE
fix(core):  Text when capturing an alterac valley mine is the wrong way around 

### PR DESCRIPTION
**Pull request by Dev-Priest**
## Changes Proposed:
-  Fix text when capturing an alterac valley mine which was the wrong way

## Issues Addressed:

-  #13374 

## Tests Performed:

- Before fix (Reproducing bug) : 
.debug bg
queue AV
capture irondeep mine
--> The text is in the incorrect way

- After fix (Testing fix) : 
.debug bg
queue AV
capture irondeep mine
--> The text is now in the correct way

## How to Test the Changes:

1. .debug bg
2. queue AV
3. capture irondeep mine

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Nefertumm — APPROVED** [review]

**Comment by Nefertumm** [comment]
Hey! Thank you for your first contribution! We will wait until it's tested before it gets merged 💯

**Comment by heyitsbench** [comment]
Tested. [Before PR (Alliance)](https://user-images.githubusercontent.com/38561765/195966761-d65217dc-c92a-4d68-a0dc-3e09388ec46b.png), [with PR (Alliance)](https://user-images.githubusercontent.com/38561765/195966859-3806e4b7-ccba-4071-af11-a5e33074d4fe.png), [with PR (Horde)](https://user-images.githubusercontent.com/38561765/195967011-5eb1378c-f7e8-4400-8786-cf8b94e40c68.png). Works well, great work!

Workflow appears to be failing due to a dependency issue unrelated to the PR.

**Comment by Kitzunu** [comment]
Thanks for the PR =)

**Pull request merged**
